### PR TITLE
Added On Dispose Info

### PR DESCRIPTION
**Pull request by pinkeshmars**
### Description
Added On Dispose Info

[Linear ticket](https://linear.app/flutterflow/issue/DEVR-715/[docs]-ondispose-action-trigger) and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-715 

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [ ] Enhancement to current docs
- [ ] Removed outdated references
- [ ] Update assets

### CODE REVIEW

File: docs/resources/ui/pages/page-lifecycle.md [hunk 1]
```diff
@@ -150,6 +150,18 @@ For instance, if you have a shortcut assigned to the letter "C" and a user tries
 To handle this, you can enable the option on the `TextField` widget to bypass keyboard shortcuts. However, it’s generally better to assign more unique combinations, like Cmd + C, which are less likely to conflict with normal typing in a text field.
 :::
 
+### On Dispose [Action Trigger]
+
+The **On Dispose** action trigger allows you to define actions that execute when a page is navigated away from or removed from memory. It is particularly useful for performing cleanup tasks, such as closing long-lived connections, canceling subscriptions, or releasing resources, to prevent memory leaks and ensure efficient resource management.
+
+:::tip[Possible Use Cases]
+
+- **Cleaning Up Resources:** Use this action trigger to cancel timers, close database connections, or unsubscribe from streams to prevent memory leaks and unnecessary processing.
```
Comment (PoojaB26) [inline docs/resources/ui/pages/page-lifecycle.md:159]: Only first example makes sense. The rest are in relation to Flutter.

**Review by PoojaB26 — CHANGES_REQUESTED** [review]
Some of the examples dont work for FlutterFlow because if the creation of them is not allowed in FF, the disposal of them wont make sense to users. We must remember the distinction between Flutter and FlutterFlow in such examples so we dont confuse the users.

**Comment by pinkeshmars** [comment]
@PoojaB26 you can take a look again.

File: docs/resources/ui/components/component-lifecycle.md [hunk 2]
```diff
@@ -71,6 +71,15 @@ created.
 Your component can also respond to certain keypress events. For more details on setting this up, see [this section on 
 keyboard shortcuts](/resources/ui/pages/page-lifecycle#on-shortcut-press-action-trigger).
 
+### On Dispose [Action Trigger]
+The **On Dispose** action trigger for components allows you to define actions that execute when the page containing the component is navigated away or removed from memory. It is particularly useful for stopping ongoing operations.
+
+Imagine a scenario where a [periodic action](../../../resources/control-flow/time-based-logic/periodic-action.md), such as fetching live weather updates, is started in a component when it is loaded (i.e, [On Initialization](#on-initialization-action-trigger)). The action runs periodically, providing real-time data updates as long as the component is active. However, when the page containg the component is navigated away, you need to stop the periodic action to conserve resources and prevent unnecessary processing. By using the On Dispose action trigger, you can safely stop the periodic updates and clean up any associated resources.
```
Comment (PoojaB26) [inline docs/resources/ui/components/component-lifecycle.md:77]:
```suggestion
Imagine a scenario where a [periodic action](../../../resources/control-flow/time-based-logic/periodic-action.md), such as fetching live weather updates, is started in a component when it is loaded (i.e, [On Initialization](#on-initialization-action-trigger)). The action runs periodically, providing real-time data updates as long as the component is active. However, when the page containing the component is navigated away, you need to stop the periodic action to conserve resources and prevent unnecessary processing. By using the On Dispose action trigger, you can safely stop the periodic updates and clean up any associated resources.
```

**Review by PoojaB26 — APPROVED** [review]
Typo fix required. Thanks for working on these fixes!

**Pull request merged**